fixtures/uking/mini-spider-light.json

```diff
@@ -0,0 +1,317 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Mini Spider Light",
+  "shortName": "UK-MiniSpider",
+  "categories": ["Effect"],
+  "meta": {
+    "authors": ["Anonymus", "CGBassPlayer"],
+    "createDate": "2023-03-21",
+    "lastModifyDate": "2023-04-24"
+  },
+  "comment": "Mini 60W Spider Light",
+  "links": {
+    "productPage": [
+      "https://www.uking-online.com/product/uking-zq-b20-mini-60w-spider-light/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=4KL1MdtnQ34"
+    ],
+    "manual": [
+      "https://www.uking-online.com/wp-content/uploads/2020/10/ZQ-B20.pdf"
+    ]
+  },
+  "physical": {
+    "dimensions": [280, 150, 130],
+    "weight": 1.99,
+    "power": 80,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "LED"
+    },
+    "lens": {
+      "degreesMinMax": [3, 3]
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "capability": {
+        "type": "Pan",
+        "angleStart": "0deg",
+        "angleEnd": "135deg"
+      }
+    },
+    "Pan 2": {
+      "name": "Pan",
```

Comment:
Those two channels should not have the same name, and the movement is more Tilt than Pan. Please rename them to "Tilt 1" and "Tilt 2" instead and change the capability types.

Reply:
That was something leftover from editor submission. Didn't really know what to do with it so thanks for the guidance there